packaging/pom.xml

```diff
@@ -95,6 +194,25 @@
         <groupId>com.akathist.maven.plugins.launch4j</groupId>
         <artifactId>launch4j-maven-plugin</artifactId>
         <version>1.7.25</version>
+
+            <!-- This is a clean snapshot of launch4j's master branch,
```

Comment:
Nice. OK, this makes sense now.  Was wondering the WHY, but this comment from you tells me that in the detail I was curious about. Thanks for this comment!  Let's do more of this!

Reply:
Testing your provided binary now, just downloaded...

Reply:
`openrefine.exe` started it just fine now and that was with my JAVA_HOME env variable pointing to Java 8.

One curious thing however that is different now... hitting CTRL-C in the console is exiting immediately and doesn't seem to be doing the Project Save prior to the shutdown?
Let me try other Java versions and also using the `refine.bat` and testing a few other things.

Reply:
If I don't have Java installed...when starting `openrefine.exe` it opens my browser and takes me to https://www.java.com/en/download/
Perhaps this needs to be changed in the future after December 2020 when Oracle finally stops Java 8 patches?
or we can change it now to something better like https://jdk.java.net ?

It works fine with Java 11 and 12 where I downloaded and use OpenJDK and set my JAVA_HOME.
![Annotation 2019-07-23 215914](https://user-images.githubusercontent.com/986438/61761748-47550f80-ad95-11e9-86f7-2e6ce8b7a7f5.png)

Reply:
About changing to https://jdk.java.net/, the main downside I can see is that users will download a zip file and will have to set up their JAVA_HOME manually, whereas Oracle's Java comes with an installer.

I think it should not be a problem for us if Oracle no longer provides Java 8 support - people will just download newer versions of Java, which should be compatible with OpenRefine. Java 12 works fine for now except for Jython deprecation warnings, but that should be fixed by the upcoming release of Jython, which should hopefully happen before the cutoff date for Java 8.

Reply:
Yes, I saw the ease of use with Oracle's installer but also opinionated for power users like me.  That's fine, the world needs easy installers as well as manual installation options.  Having both is a good thing in the Java ecosystem on Windows.

Hopefully the question of WHICH Java and WHERE to download gets a bit easier as time moves on next year.  God I hope so!

I am doubtful about Jython progress, its not moving and has not for quite some time because of lack of funding.  GraalVM might make it unnecessary by 2020...we shall see.